wdl/workflows/mixed/qc.wdl

```diff
@@ -37,6 +38,7 @@ workflow qc {
     call preseq.get_preseq_metrics as nuclear_preseq {
         input:
             nuclear_bam=nuclear_bam,
+            preseq_defects_mode=select_first([preseq_defects_mode,false]),
```

Comment:
Hmm, maybe make `preseq_defects_mode` input optional in `get_preseq_metrics` and then run `select_first` with the default there. It's weird that we're having to specify default of `false` multiple times.